wicked_pdf.gemspec

```diff
@@ -13,25 +13,25 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/mileszs/wicked_pdf'
   spec.license       = 'MIT'
   spec.date          = Time.now.strftime('%Y-%m-%d')
-  spec.description   = <<desc
-Wicked PDF uses the shell utility wkhtmltopdf to serve a PDF file to a user from HTML.
-In other words, rather than dealing with a PDF generation DSL of some sort,
-you simply write an HTML view as you would normally, and let Wicked take care of the hard stuff.
-desc
+  spec.description   = <<DESC.gsub(/^\s+/, '')
+    Wicked PDF uses the shell utility wkhtmltopdf to serve a PDF file to a user from HTML.
+    In other words, rather than dealing with a PDF generation DSL of some sort,
+    you simply write an HTML view as you would normally, and let Wicked take care of the hard stuff.
+DESC
 
   spec.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
-  
-  spec.requirements << "wkhtmltopdf"
+
+  spec.requirements << 'wkhtmltopdf'
 
   spec.add_dependency 'activesupport'
 
   spec.add_development_dependency 'rails'
   spec.add_development_dependency 'bundler', '~> 1.3'
   spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'rubocop' if RUBY_VERSION >= '2.0.0'
+  spec.add_development_dependency 'rubocop', '~> 0.50' if RUBY_VERSION >= '2.0.0'
```

Comment:
Probably want to pin to patch level since there are major changes between minors.